pkg/BUILD

```diff
@@ -83,7 +88,247 @@ pkg_zip(
         ":wkt_protos_files",
     ],
     package_file_name = "protoc-{version}-{platform}.zip",
-    package_variables = ":protoc_pkg_naming",
+    package_variables = ":protobuf_pkg_naming",
+)
+
+################################################################################
+# Source distribution packaging
+################################################################################
+
+pkg_filegroup(
```

Comment:
Can we make a TODO to delete the other way we package and use this instead?

Reply:
Done.